eng/Version.Details.xml

```diff
@@ -61,10 +57,6 @@
       <Uri>https://github.com/dotnet/aspnetcore-tooling</Uri>
       <Sha>cc096c2460701eb1072359d939628c0b91c610a1</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.WindowsDesktop.App" Version="5.0.0-preview.1.20122.1">
```

Comment:
This appears to be in use, at least for testing:

https://github.com/dotnet/sdk/blob/810f8190d6728e313a616f91a6097f7e52fb703f/eng/Versions.props#L123
https://github.com/dotnet/sdk/blob/810f8190d6728e313a616f91a6097f7e52fb703f/src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj#L108
https://github.com/dotnet/sdk/blob/810f8190d6728e313a616f91a6097f7e52fb703f/src/Layout/redist/targets/BundledSdks.targets#L9

Is there any reason this needs to be deleted?

Reply:
Looks like the version property wasn't getting updated though, does that package come out of a different repo now or what?

Thinking about it more though, I think that package dependency is used to build the product.  It doesn't look like it's being used in core-sdk.

Reply:
The names don't match: Microsoft.WindowsDesktop.App -> MicrosoftWindowsDesktopAppPackageVersion.

There does appear to be a node for that in the version details file:

```
    <Dependency Name="Microsoft.NET.Sdk.WindowsDesktop" Version="5.0.0-preview.1.20120.7" CoherentParentDependency="Microsoft.WindowsDesktop.App">
      <Uri>https://github.com/dotnet/wpf</Uri>
      <Sha>4f541e550154be4447b4c2a99b53823fcc766882</Sha>
    </Dependency>
```

Reply:
I think there is a NuGet package with the ID Microsoft.NET.Sdk.WindowsDesktop which we depend on.

If you want to try removing this dependency, you should also remove the out-of-sync property from the version.props file.  I think the build would then fail though.

Reply:
That one still exists:

https://github.com/dotnet/sdk/blob/d32f61fb3cf79a213774923fd59be3ebb0042f01/eng/Version.Details.xml#L60

and matches with the version in version.props:

https://github.com/dotnet/sdk/blob/d32f61fb3cf79a213774923fd59be3ebb0042f01/eng/Versions.props#L124

The one that I removed is a different windowsdesktop dependency that was unused (note: Microsoft.NET.**Sdk**.WindowsDesktop in the one you linked)

Reply:
I see.  I was accidentally browsing the files in master, and saw that it still had an alpha version number so I thought it wasn't being updated correctly.